web/components/issues/issue-layouts/calendar/calendar.tsx

```diff
@@ -129,6 +160,42 @@ export const CalendarChart: React.FC<Props> = observer((props) => {
               />
             )}
           </div>
+
+          {/* mobile view */}
+          <>
+            <div className="md:hidden">
```

Comment:
Also this looks like it could be a separate component all together. could you check if we can reuse the issue list and quick add form inside Daytile rather than redoing it here